core/templates/css/oppia.css

```diff
@@ -4649,6 +4648,22 @@ div.oppia-issues-learner-action {
 
 .mat-card-content.oppia-profile-portfolio-container {
   background-color: transparent;
+  /* Increase no. of columns on contributors profile page */
+  display: grid;
+  grid-template-columns: max-content max-content max-content;
+}
+
+/* Increase no. of columns on contributors profile page */
```

Comment:
Unneeded comment IMO, also I really do not understand what are you trying to say.